helm-chart-sources/pulsar/values.yaml

```diff
@@ -1501,9 +1501,9 @@ zoonavigator:
       cpu: 0.1
   image:
     repository:
-      web: elkozmon/zoonavigator-web
-      api: elkozmon/zoonavigator-api
-    tag: 0.6.0
+      web: elkozmon/zoonavigator
+      api: elkozmon/zoonavigator
```

Comment:
I wonder if we can collapse the two images. I'll need to take a closer look at the release notes to understand the difference between these two images, since they appear to be collapsed now.

Reply:
make sense